### ISSUE TITLE
Use Django's syndication framework instead of deprecated of werkzeug

### ISSUE THREAD

**Issue by chris34** [inyokaproject/inyoka#1071]
With [0.15.0](https://werkzeug.palletsprojects.com/en/0.15.x/changes/#version-0-15-0) the contrib module of werkzeug was marked as deprecated. From there Inyoka uses the `atom` module.
The simplest solution is to migrate to [Django's builtin syndication feed framework](https://docs.djangoproject.com/en/1.8/ref/contrib/syndication/).

-----

btw. if you also wonder, where we use werkzeug at all:

```
inyoka/utils/feeds.py
7:    :mod:`werkzeug.contrib.atom`.
16:from werkzeug.contrib.atom import AtomFeed

inyoka/utils/local.py
9:        a werkzeug local object bound to the current request.
20:from werkzeug import Local, LocalManager

inyoka/wiki/management/commands/generate_static_wiki.py
29:from werkzeug import url_unquote

inyoka/wiki/models.py
96:from werkzeug import cached_property
97:from werkzeug.utils import secure_filename

inyoka/forum/models.py
34:from werkzeug.utils import secure_filename

inyoka/wsgi.py
7:    the application is a `debugged Werkzeug application
8:    <http://werkzeug.pocoo.org/docs/debug/>`_.
18:    from werkzeug.debug import DebuggedApplication

inyoka/portal/models.py
16:from werkzeug import cached_property
```

 * `cached_property` also [exists in Django](https://docs.djangoproject.com/en/1.11/ref/utils/#django.utils.functional.cached_property) – the question is whether they both do the same 
 * instead of `DebuggedApplication` Django has a similar equivalent
 * for `Local`, `LocalManager` and `secure_filename` i spontanously know no alternative to

TL;DR: Either we can copy some parts (but will we get to know about CVEs in these then?) or we can also just use less of werkzeug. Latter should make updating
 easier.
